.github/workflows/bump-cnspec.yml

```diff
@@ -0,0 +1,27 @@
+name: Trigger cnquery bump in cnspec
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'cnquery version to use'
+        type: string
+        required: false
+        default: "v9.14.0"
```

Comment:
Should this not be empty? I also feel that the workflow dispatch should be in cnspec not in cnquery

Reply:
Good point.

We could use the `workflow_dispatch` to test the complete workflow, but then I'd need to rewrite this workflow. I'll remove it for now.